nautobot_netbox_importer/generator/source.py

```diff
@@ -70,6 +71,16 @@ class PreImportResult(Enum):
     USE_RECORD = True
 
 
+class SourceFieldSource(Enum):
```

Comment:
I don't get the purpose of this class. what does it mean?

Reply:
"Source Field Source" doesn't resonate with me...

Reply:
Added description for individual enum entries